Formula/fx.rb

```diff
@@ -1,10 +1,8 @@
-require "language/node"
-
 class Fx < Formula
-  desc "Command-line JSON processing tool"
-  homepage "https://github.com/antonmedv/fx"
-  url "https://registry.npmjs.org/fx/-/fx-20.0.2.tgz"
-  sha256 "7ec01246c8291cd6194587e4fe0eba92a554336ec2342a74c9eb47cf1b41179c"
+  desc "Terminal JSON viewer"
+  homepage "https://fx.wtf"
+  url "https://github.com/antonmedv/fx/archive/refs/tags/22.0.10.tar.gz"
```

Comment:
Is this the right URL? This is still 22.0.10.

Reply:
Again forgot to `git add`. Fixed.

Reply:
I'm going to implement CI to automate it.

Reply:
Is it enough?
https://github.com/antonmedv/fx/blob/master/.github/workflows/release.yml#L84-L92